Do not run ews by default on Draft PRs

aj062 opened the pull request:
#### 3805ca17be94a7a165c8d4a642455c533b4dda8f
<pre>
Do not run ews by default on Draft PRs
<a href="https://bugs.webkit.org/show_bug.cgi?id=248044">https://bugs.webkit.org/show_bug.cgi?id=248044</a>
rdar://problem/102649857

Reviewed by NOBODY (OOPS!).

* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:
(PullRequest.create_pull_request):
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/3805ca17be94a7a165c8d4a642455c533b4dda8f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [  ~~🧪 style~~](https://ews-build.webkit.org/#/builders/6/builds/104102 "The change is no longer eligible for processing.") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/13193 "Built successfully") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/37024 "The change is no longer eligible for processing.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/113311 "The change is no longer eligible for processing.") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/173614 "Built successfully and passed tests") 
| [  ~~🧪 bindings~~](https://ews-build.webkit.org/#/builders/11/builds/108043 "The change is no longer eligible for processing.") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/14259 "Built successfully") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/85/builds/4109 "The change is no longer eligible for processing.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/96331 "The change is no longer eligible for processing.") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/112382 "Built successfully") 
| [  ~~🧪 webkitperl~~](https://ews-build.webkit.org/#/builders/19/builds/109872 "The change is no longer eligible for processing.") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/10987 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/37024 "The change is no longer eligible for processing.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/96331 "The change is no longer eligible for processing.") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/5/builds/107592 "Passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/92838 "Passed tests") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/37024 "The change is no longer eligible for processing.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/96331 "The change is no longer eligible for processing.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/81/builds/6560 "The change is no longer eligible for processing.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/37024 "The change is no longer eligible for processing.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/6701 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/3576 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/20/builds/102972 "Passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/80/builds/12702 "The change is no longer eligible for processing.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/37024 "The change is no longer eligible for processing.") | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/79/builds/8480 "The change is no longer eligible for processing.") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

JonWBedard commented on Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:
```diff
@@ -442,7 +442,9 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             labels = pr_issue.labels
             did_remove = False
-            for to_remove in cls.MERGE_LABELS + cls.UNSAFE_MERGE_LABELS + ([cls.BLOCKED_LABEL] if unblock else []) + [cls.SKIP_EWS_LABEL]:
+            labels_to_remove = cls.MERGE_LABELS + cls.UNSAFE_MERGE_LABELS + ([cls.BLOCKED_LABEL] if unblock else []) + ([cls.SKIP_EWS_LABEL] if not existing_pr.draft else [])
```
I think we want to provide a `--no-ews`/`--ews` flag to `git-webkit` here too...I would expect the default to be `ews=True` for most PRs, but `ews=False` for draft PRs.

aj062 committed:
Do not run ews by default on Draft PRs
https://bugs.webkit.org/show_bug.cgi?id=248044
rdar://problem/102649857

Reviewed by NOBODY (OOPS!).

* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:
(PullRequest.create_pull_request):
